src/models/common/training_data_prep.py

```
@@ -181,3 +128,74 @@ def prepare_common_training_data(
+) -> Tuple[pd.DataFrame, pd.Series]:
+    """Remove outliers from target variable using quantile trimming."""
+    try:
+        logger.info("Removing outliers from target variables (quantile trim)...")
+        q_low, q_high = outlier_quantiles
+        y_lo = y_train.quantile(q_low)
+        y_hi = y_train.quantile(q_high)
+        keep_mask = (y_train >= y_lo) & (y_train <= y_hi)
+        x_train_filtered = x_train[keep_mask]
+        y_train_filtered = y_train[keep_mask]
+        logger.info(
+            f"Target outlier removal: {int(len(keep_mask))} : {int(keep_mask.sum())} samples ({int(len(keep_mask) - int(keep_mask.sum()))} removed"
```

Ruff (0.14.4)

Value being cast to `int` is already an integer

Remove unnecessary `int` call

(RUF046)